src/finbars.js

```diff
@@ -105,7 +105,10 @@ function FinBar(options) {
         }
     }
 
-    cssInjector(cssFinBars, 'finbar-base', options.cssStylesheetReferenceElement);
+    // don't inject stylesheets during unit tests
+    if (window && !window.finbarsIgnoreStylesheets) {
```

Comment:
Sorry for this hack, but the cssInjector() was breaking unit tests.
You'll probably have a better solution for this.